airflow/providers/amazon/aws/operators/batch.py

```diff
@@ -428,31 +431,40 @@ def __init__(
         unmanaged_v_cpus: int | None = None,
         service_role: str | None = None,
         tags: dict | None = None,
-        max_retries: int | None = None,
+        poll_interval: int = 30,
+        max_retries: int = 120,
```

Comment:
This is breaking change. If a user passes None today, it will break at next release

Reply:
I think we have handled this as non-breaking so far in other PRs 🤔 
my understanding is that None is here only for the default, and like, who would pass an explicit `None` in their right mind ?
But yeah, I guess every change is a breaking change, I can add a warning around this if you think it's necessary

Reply:
I agree but if a user does it, their DAGs will break by upgrading Amazon provider package. I think @eladkal would consider this change as breaking change and will create a new major version